src/CarbonAware.DataSources/CarbonAware.DataSources.ElectricityMaps/test/Configuration/ServiceCollectionExtensionTests.cs

```diff
@@ -18,6 +18,9 @@ public class ServiceCollectionExtensionTests
     private readonly string TokenKey = $"DataSources:Configurations:ElectricityMapsTest:APIToken";
     private readonly string DefaultTokenValue = "myDefaultToken123";
     private readonly string UseProxyKey = $"DataSources:Configurations:ElectricityMapsTest:Proxy:UseProxy";
+    private readonly string ProxyUrl = $"DataSources:Configurations:ElectricityMapsTest:Proxy:Url";
```

Comment:
nit: Convert these `private readonly string` to `const string`. These are not initialized in the constructor,

Reply:
Done, that makes sense!